README.md

```diff
@@ -31,12 +31,29 @@ Pythoshop complements existing image processing libraries in the Python ecosyste
 ## Installation
 
 ```bash
+# you can't use this for now since the package hasn't be published yet.
 $ pip install pythoshop
 ```
```

Comment:
lets add the conda, installation and poetry installation instructions too to run package ?